python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -145,7 +149,7 @@ def _common_submit(self, dirac_script):
         self.statusInfo = ''
         j.been_queued = False
         dirac_cmd = """execfile(\'%s\')""" % dirac_script
-        result = execute(dirac_cmd)
+        result = execute(dirac_cmd, cred_req=self.credential_requirements)
```

Comment:
In the new system, Dirac's `execute()` will *always* require an `ICredentialRequirement` object which must be provided by the calling scope so I can't see an easy way of wrapping this. Unless I'm missing something?